source/_integrations/rflink.markdown

```diff
@@ -219,3 +219,163 @@ This will give you output looking like this:
+      required: true
+      type: map
+      keys:
+        name:
+          description: Name for the device.
+          required: false
+          default: RFLink ID
+          type: string
+        type:
+          description: "Override automatically detected type of the light device, can be: switchable, dimmable, hybrid or toggle. See [Light Types](/integrations/light.rflink/#light-types) below."
+          required: false
+          default: switchable
+          type: string
+        aliases:
+          description: Alternative RFLink ID's this device is known by.
+          required: false
+          type: [list, string]
+        group_aliases:
+          description: "`aliases` which only respond to group commands."
+          required: false
+          type: [list, string]
+        no_group_aliases:
+          description: "`aliases` which do not respond to group commands."
+          required: false
+          type: [list, string]
+        fire_event:
+          description: Fire a `button_pressed` event if this device is turned on or off.
+          required: false
+          default: false
+          type: boolean
+        signal_repetitions:
+          description: Repeat every RFLink command this number of times.
+          required: false
+          default: 1
+          type: integer
+        group:
+          description: Allow light to respond to group commands (ALLON/ALLOFF).
+          required: false
+          default: true
+          type: boolean
+{% endconfiguration %}
+
+### Light state
+
+Initially the state of a light is unknown. When the light is turned on or off (via frontend or remote) the state is known and will be shown in the frontend.
+
+Sometimes a light is controlled by multiple remotes, each remote has its own code programmed in the light. To allow tracking of the state when switched via other remotes add the corresponding remote codes as aliases:
+
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: rflink
+    devices:
+      newkaku_0000c6c2_1:
+        aliases:
+          - newkaku_000000001_2
+          - kaku_000001_a
+```
+
+Any on/off command from any alias ID updates the current state of the light. However when sending a command through the frontend only the primary ID is used.
+
+### Light types
+
+Light devices can come in different forms. Some only switch on and off, other support dimming. Dimmable devices might not always respond nicely to repeated `on` command as they turn into a pulsating state until `on` is pressed again (for example KlikAanKlikUit). The RFLink integration support three types of lights to make things work in every situation:
+
+- *Hybrid*: This type sends a `dim` followed by an an `on` command; and `off` commands. This will make dimmable devices turn on at the requested dim level and on/off devices on. One caveat is this type is not compatible with signal repetition as multiple `on` signals will cause dimmers to go into disco mode.
```

Comment:
```suggestion
- *Hybrid*: This type sends a `dim` followed by an `on` command; and `off` commands. This will make dimmable devices turn on at the requested dim level and on/off devices on. One caveat is this type is not compatible with signal repetition as multiple `on` signals will cause dimmers to go into disco mode.
```